Formula/ibazel.rb

```diff
@@ -0,0 +1,88 @@
+class Ibazel < Formula
+  desc "Tools for building Bazel targets when source files change"
+  homepage "https://github.com/bazelbuild/bazel-watcher"
+  url "https://github.com/bazelbuild/bazel-watcher/archive/refs/tags/v0.20.0.tar.gz"
+  sha256 "7b0f98006b32d2ad5eee38eb9b363005968abf88d235e31fc6dd2a8c4336f33d"
+  license "Apache-2.0"
+
+  depends_on "bazel" => [:build, :test]
+  depends_on "go" => [:build, :test]
+  depends_on xcode: :build
+
+  # Patch to use Homebrew's Go
+  patch :DATA
```

Comment:
This would be permanent patch to make the build work with homebrew's go sdk

Reply:
It would need to be upstreamed nonetheless. If it can't work with Homebrew dependencies it should not be in core

Reply:
- actually this is addressed by rules_go now, https://github.com/bazelbuild/rules_go/issues/3302 (https://github.com/bazelbuild/rules_go/pull/3303)